src/APIConsumer.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@chainlink/contracts/src/v0.8/ChainlinkClient.sol";
+
+/**
+ * THIS IS AN EXAMPLE CONTRACT WHICH USES HARDCODED VALUES FOR CLARITY.
+ * PLEASE DO NOT USE THIS CODE IN PRODUCTION.
+ */
+contract APIConsumer is ChainlinkClient {
+    using Chainlink for Chainlink.Request;
+
+    uint256 public volume;
+    address private oracle;
+    bytes32 private jobId;
+    uint256 private fee;
+
+    event DataFullfilled(uint256 volume);
+
+    /**
+     * Network: Kovan
```

Comment:
We'd probably want to remove this comment right?

Reply:
Will do 👍